www/lendingbot.js

```diff
@@ -370,6 +376,30 @@ function doSave() {
     });
     localStorage.setItem('timespanNames', JSON.stringify(timespanNames))
 
+    // Bitcoin Display Unit
+    localStorage.displayUnitText = $('input[name="btcDisplayUnit"]:checked').val();
```

Comment:
What are you trying to do? That'd be like going:

`jQuery.displayUnitText = something`

Just do

`displayUnitText = something`